tests/devices/data/block_sda_ssd.xml.license

```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: Red Hat, Inc.
+SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
reuse does not support xml?

Reply:
It does, but python xml parser does not omit comment tags, and so `xml` files located under the `tests` folder made actual tests to fail.

This is the easiest way to handle it that does not require adapting the expectations of many tests.